src/include/OSL/strdecls.h

```diff
@@ -141,3 +141,4 @@ STRDECL("CIE", CIE)
 STRDECL("AdobeRGB", AdobeRGB)
 STRDECL("colorsystem", colorsystem)
 STRDECL("unknown", unknown)
+STRDECL("ERROR: Unknown color space transformation \"%s\" -> \"%s\"\n", ErrorColorTransform)
```

Comment:
Can you add a line break here, this one line is causing the clang-format test to fail